tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DynamicConfigurationTests.cs

```diff
@@ -204,7 +197,7 @@ static string FlattenJsonArray(JToken json)
             // json["data_streams_enabled"]?.Value<bool>().Should().Be(expectedConfig.DataStreamsEnabled);
             FlattenJsonArray(json["header_tags"]).Should().Be(expectedConfig.TraceHeaderTags ?? string.Empty);
             // FlattenJsonArray(json["service_mapping"]).Should().Be(expectedConfig.ServiceNameMapping ?? string.Empty);
-
+            json["tags"]?.ToString(Formatting.None).Should().Be(expectedConfig.GlobalTags ?? "[]");
```

Comment:
Isn't it supposed to be `"tracing_tags"` (based on `DynamicConfigConfigurationSource` 🤔 )

Reply:
+1

Reply:
Sorry I know this test is very confusing 😄  `json` is the parsed json from the diagnostic log: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/TracerManager.cs#L398-L399